### DISCUSSION TITLE
Single-line Stat Blocks

### DISCUSSION THREAD

**Discussion by wyrmisis** [Obsidian-TTRPG-Community/fantasy-statblocks#123] [Q&A]
Hi there! I'm working on putting together layouts for Old School Essentials, and I've hit a stumbling block.

In OSE (and other similar riffs on older editions of D&D/AD&D), creature stats were generally simple enough to be represented by a line of text, as opposed to a big, chunky block. They typically follow this format:

```markdown
### Creature Name
---
**AC** 9 (10), **HD** 1, **Att** Blurb on attack routine, **THAC0** 19 (+0) [...and so on]
```

I've tried building my own statline from the Text block (which does not have a callback, so that was a nonstarter), as well as the Property block (which formats my callback output as a code block). I've also tried using an Inline Group and a bunch of Property blocks, but the content doesn't flow like I'd expect inline content to do.

I'd like to be able to use the same data for these short-form stat lines as I do in my longer-form, more descriptive stat blocks. How can I go about doing so?

**Comment by sigrunixia** [comment]
This is possible within Statblocks, but it would need a combination of CSS and some ingenuity to do. 

For example, take a look at the monsters in the [Bestiary section here](https://github.com/ObsidianTTRPGProject/ObsidianTTRPGShare/tree/main/Pathfinder/Second_Edition), and how we set up their traits to include multiple "rolls" to overcome some statblock limitations.
